package.json

```diff
@@ -111,6 +112,7 @@
     "semver": "^5.1.1",
     "silent-error": "^1.0.0",
     "sort-package-json": "^1.4.0",
+    "strip-ansi": "^4.0.0",
```

Comment:
it seems like the right place (since `strip-ansi` was just extracted from `chalk` and we depended on `chalk`); we use styles from `strip-ansi` to print help out